casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala

```diff
@@ -1601,7 +1601,7 @@ class HashSetCasperTest extends FlatSpec with Matchers {
       _               <- nodes(0).receive()
       _               <- nodes(2).receive()
 
-      _     <- nodes(0).casperEff.lastFinalizedBlock shouldBeF genesisWithEqualBonds
+      _     <- nodes(0).casperEff.lastFinalizedBlock shouldBeF block1
```

Comment:
My mistake was thinking that block1 was genesis. I wish I had thought through this a bit more the first time.

Reply:
Note this isn't block2 because there are actually 4 validators involved and so there needs to be 3 total to make a majority.